UdpHosts/GameServer/Packets/GSS/Character/BaseController/MovementInput.cs

```diff
@@ -50,10 +50,10 @@ public class MovementInput
         public Vector AimDirection;
 
         [Field]
-        public ushort UnkUShort3;
+        public ushort UnkUShort3; // JetpackEnergy
 
         [Field]
-        public ushort UnkUShort4;
+        public ushort UnkUShort4; // GroundTimePositiveAirTimeNegative
 
```

Comment:
I assume this and other already switched files are obsolete now and could be removed, correct?

Reply:
Yes, but I've noticed some differences between `AeroMessages` and PIN.
Some known fields in PIN are unknown in `AeroMessages`, I'll build a list of files to check afterwards.
They can be removed without further concerns.

Reply:
Resolved in #[7d5c351](https://github.com/themeldingwars/PIN/pull/16/commits/7d5c351f5703e74b65770a39714cf1fc0f81753e)